docs/docs/customization/form.md

```diff
@@ -0,0 +1,181 @@
+Pathfora allows for some customization on form elements including hiding specific input fields, setting placeholder text, and selecting which fields are required for the user to submit the form.
+
+## fields
+
+Select which fields should be a part of the module's form.
+
+<table>
+	<tr>
+		<th>VALUE</th>
+		<th>TYPE</th>
+		<th>BEHAVIOR</th>
+	</tr>
+	<tr>
+		<td>fields</td>
+		<td>obj</td>
+		<td>must be object formatted using following options / values</td>
+	</tr>
+</table>
+
+<table>
+	<tr>
+		<td colspan="3" align="center"><code>fields</code> object</td>
+	</tr>
+	<tr>
+		<th>VALUE</th>
+		<th>TYPE</th>
+		<th>BEHAVIOR</th>
+	</tr>
+	<tr>
+		<td> name </td>
+		<td>boolean</td>
+		<td><code>optional</code> Supported Types: Form, Gate</td>
+	</tr>
+	<tr>
+		<td> title </td>
+		<td>boolean</td>
+		<td><code>optional</code> Supported Types: Form, Gate</td>
+	</tr>
+	<tr>
+		<td> email </td>
+		<td>boolean</td>
+		<td><code>optional</code> Supported Types: Form, Gate</td>
+	</tr>
+	<tr>
+		<td> message </td>
+		<td>boolean</td>
+		<td><code>optional</code> Supported Types: Form</td>
+	</tr>
+	<tr>
+		<td> organization </td>
+		<td>boolean</td>
+		<td><code>optional</code> Supported Types: Gate</td>
+	</tr>
+</table>
+
+### Show/Hide Fields - [Live Preview](../../examples/preview/customization/form/fields.html)
+
+![Form Placeholders](../examples/img/customization/form/fields.png)
+
+<pre data-src="../../examples/src/customization/form/fields.js"></pre>
```

Comment:
I know I am reviewing docs here, but I again don't like the API here. Why not have fields be an array of strings? Using an object also means order can't be specified, since object key order [isn't guaranteed](http://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order) in JS.